docs/src/pages/components/data-grid/filtering/QuickFilteringGrid.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import TextField from '@material-ui/core/TextField';
+import {
+  DataGrid,
+  escapeRegExp,
```

Comment:
Are we sure about escaping the regexp? We could keep it simple...
If we do we could import an escapeRegExp func or write it in the demo, it could be simpler 🤔